tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 360x360 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#let's do some posterior predictive checks to see if the\n",
+    "#observation `x_o` remains within the support of the posterior\n",
+    "from sbi.analysis import pairplot\n",
+    "\n",
+    "fig, ax = pairplot(\n",
+    "    samples=posterior_data,\n",
+    "    points=x_o,\n",
+    "    limits=list(zip(.75*x_o.flatten(), 1.25*x_o.flatten())),\n",
+    "    upper=['kde'],\n",
+    "    diag=['kde'],\n",
+    "    figsize=(5,5),\n",
+    "    labels=[rf\"$\\theta_{d}$\" for d in range(3)]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "The posterior is well aligned with the observation. On top, the observation falls into the support of the posterior samples. That is indicative that our posterior estimate matches the true posterior very well."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Running SBC\n",
+    "\n",
+    "We have a working and trained posterior at this point! Hurray! Let's look at the SBC metrics now."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "num_sbc_runs = 1_000 #the higher sbc runs the better\n",
+    "thetas = prior.sample((num_sbc_runs,)) #we sample again from the prior (as an alternative, we could have used our holdout set)\n",
+    "xs = simulator(thetas) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "8fb9cd2fd9a24bd298fd1afae207ce7d",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Running 1000 sbc samples.:   0%|          | 0/1000 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ranks, dap_samples = run_sbc(thetas, xs, posterior, num_workers=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "`sbi` has established two methods to do simulation-based calibration:\n",
+    "\n",
+    "- metrics to compare the sbc ranks with a uniform distribution \n",
+    "- control plots for visual inspections like fig. 1 or 2 in [Talts et al, 2018](https://arxiv.org/abs/1804.06788)\n",
+    "\n",
+    "The `ranks` count per dimension of `theta` how many posterior samples fall below `theta_o`. Based on theory, the distribution of these ranks per dimension of `theta` should turn out to be uniformly distributed. \n",
```

Comment:
add one sentence in the intro that SBC ranking is performed on the 1-D marginals for each parameter separately.

Reply:
I think it is a good idea to introduce marginals here and not in the intro. We are building up level of detail towards the end anyhow.
063b9f6